File: u3u2/Assets/Scripts/app/net/s2c/GCArenaKillCd.cs

using System;
namespace app.net
{
/**
 * 消除竞技场cd成功
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCArenaKillCd :BaseMessage
{

	public GCArenaKillCd ()
	{
	}

	protected override void ReadImpl()
	{


	}
	
	protected override void WriteImpl()
    {
        return;
    }
	
	public override short GetMessageType() 
	{
		return (short)MessageType.GC_ARENA_KILL_CD;
	}
	
	public override string getEventType()
	{
		return ArenaGCHandler.GCArenaKillCdEvent;
	}
	

}
}

File: IdeaServer/zlj2019/game_server/src/com/imop/lj/gameserver/arena/msg/GCArenaKillCd.java
package com.imop.lj.gameserver.arena.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.GCMessage;
/**
 * 消除竞技场cd成功
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCArenaKillCd extends GCMessage{
	

	public GCArenaKillCd (){
	}
	

	@Override
	protected boolean readImpl() {


		return true;
	}
	
	@Override
	protected boolean writeImpl() {

		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.GC_ARENA_KILL_CD;
	}
	
	@Override
	public String getTypeName() {
		return "GC_ARENA_KILL_CD";
	}
}